compiler/modules/nback/structure.bal

```diff
@@ -31,7 +31,16 @@ final RuntimeFunction mappingIndexedSetFunction = {
 };
 
 final RuntimeFunction listConstructFunction = {
```

Comment:
This needs to be listConstruct8Function to be consistent with existing naming patterns